x-pack/plugins/security_solution/public/resolver/view/panels/process_details.tsx

```diff
@@ -51,9 +52,9 @@ export const ProcessDetails = memo(function ProcessDetails({
   const processName = event.eventName(processEvent);
   const entityId = event.entityId(processEvent);
   const isProcessTerminated = useSelector(selectors.isProcessTerminated)(entityId);
-  const processInfoEntry = useMemo(() => {
+  const processInfoEntry: EuiDescriptionListProps['listItems'] = useMemo(() => {
     const eventTime = event.eventTimestamp(processEvent);
-    const dateTime = eventTime ? formatDate(eventTime) : '';
+    const dateTime = eventTime === undefined ? null : formatDate(eventTime);
```

Comment:
By checking if the value is `undefined` we can support showing the date even if it is the unix epoch (00:00:00 UTC on 1 January 1970.)